platforms/tktrex/backend/parsers/index.ts

```diff
@@ -0,0 +1,25 @@
+import nature from './nature';
+import description from './description';
+import music from './music';
+import hashtags from './hashtags';
+import numbers from './numbers';
+import stitch from './stitch';
+import author from './author';
+import search from './search';
+import profile from './profile';
+import native from './native';
+import downloader from './downloader';
+
+export const parsers = {
+  nature,
+  description,
+  music,
+  hashtags,
+  numbers,
+  stitch,
+  author,
+  search,
+  profile,
+  native,
+  downloader,
```

Comment:
nota: downloder deve sempre andare per ultimo e nature sempre per primo. altri plugin possono avere dipendenze di priorità, pertanto sarebbe necessario garantire ordine di esecuzione

Reply:
downloder in aggiunta non "parsa" ma anche lui estende il metadata con dei valori che dicono dove sta il file condiviso.

in un futuro prossimo dovremmo avere il plugin downloader che verifica il campo `supporter` all'interno della evidence perchè solo alcuni determinati `publiKey` (o anche alcuni `experimentId` o `reseaerchTag` da `html`) potrebbero essere i trigger del download oppure no.